### PR TITLE
DNS IP custom ports for IPv4 #113993

### PR DESCRIPTION
**Pull request by HarvsG**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/113993
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Documentation**
	- Enhanced the DNS IP integration description to include examples of setting custom nameserver IP addresses and optionally specifying a port number.

### CODE REVIEW

File: source/_integrations/dnsip.markdown [hunk 1]
```diff
@@ -25,5 +25,7 @@ The **DNS IP** {% term integration %} will expose an IP address, fetched via DNS
 
 {% configuration_basic %}
 Resolver:
-  description: "You may override the default nameservers that are being used by setting any nameserver you like."
+  description: "You may override the default IPV4 and IPV6 nameservers that are being used by setting any nameserver IP address you like, for example `1.1.1.1` (IPV4) or `2606:4700:4700::1111` (IPV6)."
+Port:
+  description: "You may override the default DNS port of `53`. This may be useful to bypass DNS filtering or redirection."
```
Comment (c0ffeeca7) [inline source/_integrations/dnsip.markdown:30]: Would it be worth mentioning that this applies to IPv4 addresses only?

Comment (gjohansson-ST) [reply source/_integrations/dnsip.markdown:30]: It doesn't. It's for both ipv4 and ipv6. 
There is a resolver and a port for each but it's not duplicated here as it's only a descriptive information about the fields.

Comment (c0ffeeca7) [reply source/_integrations/dnsip.markdown:30]: Thank you for clarifying!

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @HarvsG :+1:

**Pull request merged**